debian/control

```diff
@@ -0,0 +1,165 @@
+Source: letsencrypt
+Section: web
+Priority: optional
+Maintainer: Debian letsencrypt <letsencrypt-devel@lists.alioth.debian.org>
+Uploaders: Harlan Lieberman-Berg <hlieberman@setec.io>,
+	   Francois Marier <francois@debian.org>
+Build-Depends: debhelper (>= 9),
+	       dialog,
+               dh-python,
+               python-all,
+	       python-augeas,
+               python-configargparse,
+	       python-configobj,
+               python-dialog,
+               python-docutils,
+	       python-mock,
+	       python-ndg-httpsclient,
+               python-nose,
+               python-nosexcover,
+               python-openssl (>= 0.15),
+               python-parsedatetime,
+	       python-psutil,
+	       python-pyasn1,
+	       python-pyparsing,
+               python-repoze.sphinx.autointerface,
+	       python-requests,
+               python-rfc3339,
+	       python-setuptools,
+               python-sphinx,
+               python-sphinx-rtd-theme,
+       	       python-tz,
+               python-werkzeug,
+               python-zope.component,
+	       python-zope.interface
+Standards-Version: 3.9.6
+Homepage: https://www.letsencrypt.org/
+X-Python-Version: >= 2.7
+
+Package: python-acme
+Section: python
+Architecture: all
+Depends: openssl,
+	 python-configargparse,
+	 python-cryptography (>= 0.8),
+	 python-mock,
+	 python-rfc3339,
+	 python-pyasn1,
+	 python-ndg-httpsclient,
+	 python-openssl (>= 0.15),
+	 python-tz,
+	 python-requests,
+	 python-six,
+	 python-werkzeug,
+	 ${misc:Depends},
+	 ${python:Depends}
+Description: ACME protocol library for Python 2
+ This is a library used by the Let's Encrypt client for the ACME
+ (Automated Certificate Management Environment).  The ACME protocol is
+ designed as part of the Let's Encrypt project, to make it possible to
+ setup an HTTPS server and have it automatically obtain a
+ browser-trusted certificate, without any human intervention.  This
+ library implements the protocol used for proving the control of a
+ domain.  It currently supports only Python 2.
+
+Package: letsencrypt
+Architecture: all
+Depends: dialog,
+	 openssl,
```

Comment:
openssl is not required. It can be put into `Suggests`, because manual authenticator uses it in the output (though this output might not be run on the same machine, but on the target webserver instead)